docs/devel.md

```diff
@@ -4,14 +4,32 @@ Vendoring is performed with `glide`. To avoid `k8s.io/kubernetes`vendoring, some
 
 # How to generate deep-copy
 
-First of all you need `deepcopy-gen`. You can retrieve directy from `gengo` package via a simple:
+First of all you need kubernetes `code-generator` executables. To install in your `$GOPATH` one need to run:
 
 ```shell
-$ go get go get k8s.io/gengo
+$ go get k8s.io/code-generator/...
+
 ```
 
 Now you can generate the deepcopy functions running something like this...
 
 ```shell
 $GOPATH/bin/deepcopy-gen  --v 4 --logtostderr -i github.com/sdminonne/workflow-controller/pkg/api/v1  --bounding-dirs "k8s.io/api"
 ```
+
+and the client go
+
+```shell
+$GOPATH/bin/client-gen --input-base "github.com/sdminonne/workflow-controller/pkg/api" --input "workflow/v1" --go-header-file "hack/boilerplate.go.txt"   --clientset-path "github.com/sdminonne/workflow-controller/pkg/client" --clientset-name "versioned"
+```
+
+now the listers:
+
+```shell
+${GOPATH}/bin/lister-gen --input-dirs "github.com/sdminonne/workflow-controller/pkg/api/workflow/v1" --output-package "github.com/sdminonne/workflow-controller/pkg/client/listers" -h "hack/boilerplate.go.txt"
+```
+
+now the informers:
```

Comment:
If this is a standard sequence of commands folks need to run, would it be worth scripting?

Reply:
Things are changing in kube repo. Soon there's will be a script to run all this. That's why I didn't want to write one. Goal of this is just to show how things should be run in case someone wants to do the same ;) @clamoriniere1A